### ISSUE TITLE
SGC screens controls

### ISSUE THREAD

**Issue by danc77** [RafaelDeJongh/cap#50]
can we know the controls for all the sgc screens please

**Comment by CryptAlchemy** [comment]
Here's how you use them.
** Make sure to spawn your screens with a keyboard otherwise you'll have a fun time trying to attach a keyboard to them xd
Place your screens using the tool by left-clicking.
Place your SGC computer using the tool by right-clicking.
Hold C and right-click on your SGC computer to set your codes.
To turn the system on, press e on the computer.
To turn the system off, hold e on the computer.
Press numpad 0 to switch between screens and press enter to select the screen after highlighting it using arrow keys.
To self destruct, go to the Gate Active screen and hold numpad 1; type in your code; press enter; press numpad 1 to activate the bomb.
To abort, go to the Gate Active screen and type your abort code only. Then press enter.
To dial, go to your address book screen and select one of the addresses with arrow keys and hit enter twice.
To manually dial, enter your desired address into the dialing computer screen.
To turn your screens on and off individually, press e on them.
To shut the gate down, press backspace on the dialing computer screen.

There's prolly more but that's all I can think of right now

**Comment by AlexALX** [comment]
@CryptAlchemy Thanks for taking time and write this instructions.

I can also add last thing - to scroll in galaxy map you need use "Page Up/Down" Keys and then arrows (left/right/up/down).

**Comment by danc77** [comment]
Thanks, but i've also discovered that on the chevron 7 screen if you press enter the ring starts spinning but you can't seem to be able to lock a symbol, i'm i missing something or is that just not implemented?

**Comment by johnoldiges** [comment]
Hello, I know that time is limited for cap requests and such, but I just figured I'd throw this idea out there for anyone willing to do it. On the SGC Screen there be an input on the Main Computer (Desktop) for "Base Alert Code 9" and When the input receives a signal (like "1" for button being pushed) All of the Auxiliary Screens ( chevron 7 screen and such, Not Dialing Computer and other primary screens) Switch to Displaying the Base Alert Code 9 screen. Again I know time is limited but I figured if liked it enough and had the time they may contribute the code.
